src/WeaverUser.coffee

```
@@ -1,26 +1,54 @@
 # Libs
-cuid = require('cuid')
-WeaverNode = require('./WeaverNode')
+cuid   = require('cuid')
+Weaver = require('./Weaver')
+loki   = require('lokijs')
+
+class WeaverUser
+  
+  constructor: () ->
+    @email
+    @username
+    @password
+    @emailVerified = false
+    
+    
+  logIn: (usr, pass) ->
+    credentials = {
+      user:usr
+      password:pass
+    }
+    coreManager = Weaver.getCoreManager()
+    users = Weaver.getUsersDB()
+    coreManager.logIn(credentials)
+    .then((res) ->
+      try
+        users.insert({user:usr,token:res[0].token})
+      catch error
+        console.error(error)
```

What error do you expect here? The user of the SDK might not get feedback of this error when printed. Is it not better to either let it fail or reject with some error code